image_test.go

```diff
@@ -367,6 +368,55 @@ func TestImageRW(t *testing.T) {
 	}
 }
 
+func TestImageCubeRoundTrip(t *testing.T) {
+	strides := []int{2, 3, 6} // non repretitive, aperiodic strides
+
+	// the data looks like [0, 1, 2, ...]
+	data := make([]int16, 2*3*6)
+	for i := 0; i < len(data); i++ {
+		data[i] = int16(i - 32768)
+	}
+	var b bytes.Buffer
+	fits, err := Create(&b)
+	if err != nil {
+		t.Error(err)
+	}
+	im := NewImage(16, strides)
+	cards := []Card{
+		{Name: "BZERO", Value: 32768},
+		{Name: "BSCALE", Value: 1.}}
```

Comment:
it's nitpicking at its best but...
```suggestion
		{Name: "BSCALE", Value: 1.},
	}
```